### ISSUE TITLE
Uncaught Error: ENOENT, open '/Users/non/Repositories/ohmpieng/production/config/omniauth.rb'

### ISSUE THREAD

**Issue by nonmadden** [atom/atom#4994]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.169.0
**System**: Mac OS X 10.10.2
**Thrown From**: Atom Core

### Stack Trace

Uncaught Error: ENOENT, open '/Users/non/Repositories/ohmpieng/production/config/omniauth.rb'

```
At /Applications/Atom.app/Contents/Resources/app/node_modules/pathwatcher/node_modules/q/q.js:126

Error: ENOENT, open '/Users/non/Repositories/ohmpieng/production/config/omniauth.rb'
  at Error (native)

```

### Commands

```
  4x -0:02.6 pane:show-previous-item (atom-text-editor.editor)
 11x -0:00.0 pane:show-next-item (atom-text-editor.editor)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "monokai"
    ]
  },
  "editor": {
    "fontFamily": "Menlo",
    "fontSize": 12,
    "invisibles": {}
  }
}
```

### Installed Packages

```coffee
# User
monokai, v0.8.0

# Dev
No dev packages
```

**Comment by Moter8** [comment]
```
[Enter steps to reproduce below:]

...
...
```

These things are there so you write in steps. Please do so.

**Comment by kevinsawicki** [comment]
I believe this is duplicate of https://github.com/atom/atom/issues/4850 and will be fixed by #4906
